fedora/templates/csv/sysctl_values.csv

```diff
@@ -6,3 +6,4 @@ kernel.kptr_restrict,1
 kernel.randomize_va_space,2
 kernel.yama.ptrace_scope,1
 net.ipv6.conf.all.disable_ipv6,1
+crypto.fips_enabled,1
```

Comment:
FIPS checks should never be templated as FIPS enablement should be heavily scrutinized.